libs/libc/time/lib_gmtimer.c

```diff
@@ -336,7 +336,7 @@ FAR struct tm *gmtime_r(FAR const time_t *timep, FAR struct tm *result)
   result->tm_wday   = clock_dayoftheweek(day, month, year);
   result->tm_yday   = day +
                       clock_daysbeforemonth(result->tm_mon,
-                                            clock_isleapyear(year));
+                                            clock_isleapyear(year)) - 1;
```

Comment:
maybe `day - 1 + clock_daysbeforemonth()` just for the style consistency? Or change to `tp->tm_mday + clock_daysbeforemonth() - 1` in other places?

Reply:
Yep, sorry. I did it in two steps and I didn't notice it.  
Fixed.